docs/UserGuide.md

```diff
@@ -142,37 +122,67 @@ In this section, you will learn how to set up LinkMeIn in your computer.
         <img src="images/quick-start/startupPage.png" alt="Start Up Page LinkMeIn" width="650"/>
     </div>
 
-9. Great job! You are now ready to learn about the [Command Line Interface](#using-linkmeins-cli) of LinkMeIn!
+9. Great job! You are now ready to learn about the [GUI](#introducing-linkmeins-gui) of LinkMeIn!
+
+[&uarr; Back to Table of Contents](#table-of-contents)
+
+# Introducing LinkMeIn's GUI
+LinkMeIn's [GUI](#glossary) is designed to be **simple** and **intuitive** with you in mind.
+The GUI consists of four main components:
+
+![uiIntro.png](images/ui-intro/uiIntro.png)
+
+If you are unsure about what each component does, you can refer to the table below:
+
+| Number | Component            | Description                                              |
+|--------|----------------------|----------------------------------------------------------|
+| 1      | Command Box          | Key in your commands here and LinkMeIn will execute them |
+| 2      | Message Box          | Displays success or error messages to you                |
+| 3      | Company List Panel   | Displays the list of companies that you have added       |
+| 4      | Company Detail Panel | Displays the detail of the company that you are viewing  |
```

Comment:
I think it will make sense to say "Display the details of the company that you are viewing" 

instead of detail of a company that you are viewing. Since we are referring to a specific company so it make sense to use "the" instead of "a"